nengo_gui/static/sim_control.css

```diff
@@ -122,3 +122,24 @@
     padding: 0.2em !important;
     border: none !important;
 }
+
+#speed_throttle {
+    height: 10px;
+    width: 110px;
+    #background: yellow;
```

Comment:
Oops -- that's me forgetting what language I'm using and trying to comment out a line, and then forgetting to remove it before committing.  Oops...  removing that line now....